espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/SourceDocument.java

```diff
@@ -87,6 +87,8 @@ private void buildXML(Document doc, Element parentElement, View view) {
         setAttribute(element, ViewAttributesEnum.LONG_CLICKABLE, Boolean.toString(viewElement.isLongClickable()));
         setAttribute(element, ViewAttributesEnum.CLASS, viewElement.getClassName());
         setAttribute(element, ViewAttributesEnum.INDEX, Integer.toString(viewElement.getIndex()));
+        if (viewElement.getText() != null)
+            setAttribute(element, ViewAttributesEnum.TEXT, viewElement.getText());
```

Comment:
it is recommended to enclose condition body into curly brackets even if there is only one operator.